src/main/kotlin/tanvd/grazi/ide/ui/GraziConfigurable.kt

```diff
@@ -1,12 +1,40 @@
 package tanvd.grazi.ide.ui
 
+import com.intellij.ide.ui.search.SearchableOptionsRegistrar
 import com.intellij.openapi.options.ConfigurableBase
 import tanvd.grazi.GraziConfig
+import tanvd.grazi.language.Lang
 
 class GraziConfigurable : ConfigurableBase<GraziSettingsPanel, GraziConfig>(
         "reference.settingsdialog.project.grazi", "Grazi", null) {
 
+    private lateinit var ui: GraziSettingsPanel
+
+    init {
+        // TODO search
+        val registrar = SearchableOptionsRegistrar.getInstance();
+        registrar.addOption("spellcheck", displayName, displayName, id, displayName)
```

Comment:
Create here some extension function to reduce boilerplate with `displayName, displayName, id, displayName`

Reply:
Reworked